runtime/moonriver/src/xcm_config.rs

```diff
@@ -493,8 +493,6 @@ pub enum CurrencyId {
 	SelfReserve,
 	// Assets representing other chains native tokens
 	ForeignAsset(AssetId),
-	// Our local assets
-	DeprecatedLocalAssetReserve(AssetId),
```

Comment:
Please keep the dummy variant to avoid breaking changes for CurrencyId scale encoding